### PR TITLE
ES6&TS 4

### PR DESCRIPTION
**Pull request by elinorga**

### CODE REVIEW

File: modules/05 ES6 and Typescript/Assignments/4/main.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+interface IStringUtills<T> {
+    getVowels(str: string): T;
+    removeChar(str: string, pos: T): string;
+    incrementLetters(str: string): string;
+}
+class StringUtils implements IStringUtills<number>{
+
+    public getVowels = (str: string): number => {
+        let count: number = 0;
+        let vowels: any = "aeiou";
```
Comment (JavascriptBootcamp) [inline modules/05 ES6 and Typescript/Assignments/4/main.ts]: vowels type should be "string"

File: modules/05 ES6 and Typescript/Assignments/4/main.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    return max.theString;
+}
+
+let strMaxLength: string = findLongestString('Web Development Tutorial');
+
+console.log(strMaxLength);
+
+// exercise 24
+interface IRandomWord<T> {
+    charsArray: Array<string>;
```
Comment (JavascriptBootcamp) [inline modules/05 ES6 and Typescript/Assignments/4/main.ts]: use string[] when defining an array of strings

File: modules/05 ES6 and Typescript/Assignments/4/main.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    randomize(length: T): string;
+}
+class RandomWord implements IRandomWord<number> {
+    public charsArray: Array<string>;
```
Comment (JavascriptBootcamp) [inline modules/05 ES6 and Typescript/Assignments/4/main.ts]: use string[] when defining an array of strings

File: modules/05 ES6 and Typescript/Assignments/4/main.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    constructor(charsArray: Array<string>) {
```
Comment (JavascriptBootcamp) [inline modules/05 ES6 and Typescript/Assignments/4/main.ts]: use string[] when defining an array of strings

File: modules/05 ES6 and Typescript/Assignments/4/main.ts (hunk 1 above)
Comment (elinorga) [reply modules/05 ES6 and Typescript/Assignments/4/main.ts]: I know. Just when i did a string type it somehow threw an error and asked to put vowels as any.

**Pull request closed without merging**